### PR TITLE
Unifying Side Bars and updating styling

### PR DESCRIPTION
**Pull request by salameer**
This PR Mainly contains updates to the look and feel of the website focused on the Navigation components (Sidebars). It includes:
- Unifying all side bars under one and removing language individual side bars.
- Cleaning up the .Net Sidebar section.
- Reorganizing sidebar for most languages with API documentation links.
- added missing links in the release pages.

Note next step after this PR is updates to the content on the site that are expected within a week.

You can find a live demo of these changes here:

https://guidelinescollab.github.io/azure-sdk/

You can expect to see the menu looking like these snapshots:

Screen Shot 2020-11-12 at 9 50 02 AM
Screen Shot 2020-11-12 at 9 50 02 AM

### CODE REVIEW

**Comment by salameer** [comment]
FYI @annelo-msft and @JonathanGiles

**Review by vhvb1989 — APPROVED** [review]
just a nit:
The `Latest` title can't be collapsed when clicking directly in the label (only by clicking to the little arrow works). It contains a link (maybe to refresh the page for getting the latest stuff each time, or maybe it is a bug) which prevents the collapsible menu to work.
This is a minor however.
The menu looks much better and it is now always visible while scrolling down!  nice!

**Review by czubair — APPROVED** [review]
I like it @salameer!

**Comment by salameer** [comment]
More to come @czubair  will keep you Posted on more changes and want to get your opinion.

Captain @vhvb1989 I'd like to sync up with you to get a demo on the problem and I can follow up with a fix in the upcoming changes :)

File: docs/dotnet/introduction.md [hunk 1]
```diff
@@ -64,7 +64,11 @@ The pipeline can be found in the [Azure.Core] package, and it takes care of many
 
 ## Service Client Design {#dotnet-client}
 
-Azure services will be exposed to .NET developers as one or more _service client_ types, and a set of _supporting types_. Service clients are the main starting points for developers trying to call Azure services, and each client library should have at least one client in its main namespace. The guidelines in this section describe patterns for the design of a service client.  A service client should look like this code snippet:
+Azure services will be exposed to .NET developers as one or more _service client_ types, and a set of _supporting types_. 
+
```
Comment (annelo-msft) [inline docs/dotnet/introduction.md:68]: Why remove the part about the main entry point?  It should stay at the top level, and the points about namespaces should be reiterated in the namespaces section.

Comment (salameer) [reply docs/dotnet/introduction.md:68]: Captain Anne :) I didnt remove any text, I added a namespaces header and you'll see that the text continues there : )

we will any ways have to clean this up in the next phase of content updates

Comment (annelo-msft) [reply docs/dotnet/introduction.md:68]: Cool - cleaning supplies at the ready.

**Comment by vhvb1989** [comment]
> I'd like to sync up with you to get a demo on the problem

For sure. Anyway, it's just a nit.

I found another, not-nit, issue with on the layout. 
For smaller screen, or window mode, the table doesn't have its own scrollbar, so the content ends up overriding the navigation menu if we scroll right to the content:
image

I think that doesn't happen currently because the menu is fixed position.

I suggest adding `overflow-x:auto;` to the parent div style where the tables are
image

This will also prevent tables to override any content to the right.

**Comment by salameer** [comment]
@vhvb1989 Super sorry just noticed your comment after closing :(

You're right :) This seems to be an existing issue and not necessarily a regression. would you mind opening or submitting a fix and I can work with you on it ;)

**Pull request merged**